demos/projects/ESPRESSIF/aduotaupdate/partitions_ota.csv

```diff
@@ -1,6 +1,6 @@
 # Name,   Type, SubType, Offset,   Size, Flags
 # Note: if you have increased the bootloader size, make sure to update the offsets to avoid overlap
-nvs,      data, nvs,     ,        0x4000,
+nvs,      data, nvs,     ,        0x6000,
```

Comment:
what was this expanded for?

Reply:
To allow the network stack to have enough memory.
I was getting this failure with 0x4000:
ESP_ERROR_CHECK failed: esp_err_t 0x110d (ESP_ERR_NVS_NO_FREE_PAGES) at 0x400870f0
when calling nvs_flash_init